src/app/components/MediaLoader/index.tsx

```diff
@@ -31,25 +34,99 @@ const BumpLoader = () => (
   </Helmet>
 );
 
-const MediaContainer = ({ playerConfig }: { playerConfig: PlayerConfig }) => {
+const AdvertTagLoader = () => {
+  const location = useLocation();
+  const queryString = location ? location.search : '';
+
+  useEffect(() => {
+    // Set window.dotcom to disabled if it doesn't load in 2 seconds.
```

Comment:
Is the 2 second wait an advised approach?

Reply:
I took it straight from the Morph Modules script.

Reply:
Ah cool, just found the reference. Just wanted to check it was behaviour from before.

I think if we're using it in a `useEffect`, we should do a cleanup if the user navigates away before it completes. So something like:
```
  useEffect(() => {
    // Set window.dotcom to disabled if it doesn't load in 2 seconds.
    const resolveTimeout = setTimeout(() => {
      if (window.dotcom.ads.resolves) {
        window.dotcom.ads.resolves.enabled.forEach(res => res(false));
        window.dotcom.ads.resolves.getAdTag.forEach(res => res(''));
      }
    }, 2000);

   ...rest of code

    return () => {
      clearTimeout(resolveTimeout);
    };
  }, [queryString]);
```